unresolved-package-handler/base/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - deploymentconfig.yaml
+  - imagestream.yaml
+commonLabels:
```

Comment:
please replace this by something like https://github.com/thoth-station/thoth-application/blob/master/core/base/kustomization.yaml#L4-L6